plugins/woocommerce/client/blocks/assets/js/blocks/product-elements/add-to-cart-form/edit.tsx

```diff
@@ -46,6 +47,15 @@ const AddToCartFormEdit = ( props: BlockEditProps< Attributes > ) => {
 		[]
 	);
 
+	// Check whether the current theme is block theme or not.
+	const isBlockTheme = useSelect( ( select ) => {
+		const themeSupports = select( coreStore ).getThemeSupports?.();
+		return themeSupports?.['block-templates'] === true;
+	}, [] );
+
+	const buttonBlockClass = ( ! isBlockTheme ) ? 'wp-block-button' : '' 
+	const buttonLinkClass = ( ! isBlockTheme ) ? 'wp-block-button__link wc-block-components-button' : ''
```

Comment:
We can remove the parentheses here:

```suggestion
	const buttonBlockClass = ! isBlockTheme ? 'wp-block-button' : '' 
	const buttonLinkClass = ! isBlockTheme ? 'wp-block-button__link wc-block-components-button' : ''
```